rl/wolf_sheep/agents.py

```diff
@@ -0,0 +1,78 @@
+from mesa_models.wolf_sheep.agents import GrassPatch, Sheep, Wolf
+
+from .utility import move
+
+
+class SheepRL(Sheep):
+    def step(self):
```

Comment:
Do you need to inherit this from mesa_models?  My concern is any change to that model will break this model and that inheriting the Sheep class is not necessary so it makes it unnecessarily brittle.